spec/lib/pact_broker/api/decorators/webhook_request_template_decorator_spec.rb

```diff
@@ -69,6 +75,29 @@ module Decorators
           it "reads the password" do
             expect(subject.password).to eq password
           end
+
```

Comment:
Ditto for this - all functional tests belong in the redactor spec - the only thing that goes in this spec is asserting that the redactor gets called with the right arguments.

Reply:
👍

Reply:
tests moved into `spec/lib/pact_broker/webhooks/url_redactor_spec.rb`